lib/LoginStrategy.ts

```diff
@@ -413,6 +425,27 @@ export class LoginStrategy extends BaseStrategy {
     );
   }
 
+  /**
+   * extracts siwe message and signature either from request body or query
+   * @param req
+   * @returns {string} siwe object - message and signature
+   */
+  extractSiwe(req: Request): SiweMessageReqPayload {
+    if (req.body.identity)
```

Comment:
Do you recall why message could be in `indentity` rather then in request body? Is it for legacy reason?

Reply:
I think for the old authentication protocol, we accept an object and expect the JWT to be on the `identity` prop or the prop that is specified during configuration. However, I think for siwe, we don't need to look for the `identity` prop and can directly look for `message` and `signature` in the request body.